File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--add-a-creator-method_1.vb
    Public Shared Function Create(ByVal newContact As Contact) As Contact
        Const ServerName As String = "MySQLServerName"
        Dim dataContext As AdventureWorksDataContext = _
            New AdventureWorksDataContext("Data Source=" & ServerName & _
                ";Initial Catalog=AdventureWorks;Integrated Security=True")

        Dim TempContact As New Contact()

        With TempContact
            .FirstName = newContact.FirstName
            .LastName = newContact.LastName
            .EmailAddress = newContact.EmailAddress
            .Phone = newContact.Phone
            .EmailPromotion = newContact.EmailPromotion
            .NameStyle = newContact.NameStyle
            .PasswordHash = newContact.PasswordHash
            .PasswordSalt = newContact.PasswordSalt
            .ModifiedDate = DateTime.Now
            .rowguid = Guid.NewGuid()
        End With

        dataContext.Contacts.InsertOnSubmit(TempContact)
        dataContext.SubmitChanges()
        Return TempContact

    End Function

File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--add-a-creator-method_1.cs
        public static Contact Create(Contact newContact)
        {
            const string ServerName = "MySQLServerName";
            AdventureWorksDataContext dataContext = new AdventureWorksDataContext
                  ("Data Source=" + ServerName + ";" +
                   "Initial Catalog=AdventureWorks;Integrated Security=True");

            Contact contact = new Contact();

            contact.FirstName = newContact.FirstName;
            contact.LastName = newContact.LastName;
            contact.EmailAddress = newContact.EmailAddress;
            contact.Phone = newContact.Phone;
            contact.EmailPromotion = newContact.EmailPromotion;
            contact.NameStyle = newContact.NameStyle;
            contact.PasswordHash = newContact.PasswordHash;
            contact.PasswordSalt = newContact.PasswordSalt;
            contact.ModifiedDate = DateTime.Now;
            contact.rowguid = Guid.NewGuid();

            dataContext.Contacts.InsertOnSubmit(contact);
            dataContext.SubmitChanges();
            return contact;

        }